editor/components/block-list/style.scss

```diff
@@ -422,6 +493,8 @@
 
 .editor-block-list .editor-inserter {
 	margin: $item-spacing;
+	cursor: move;/* Fallback for IE/Edge < 14 */
+	cursor: grab;
```

Comment:
Why?

Reply:
cc @chriskmnds

Reply:
It's for showing the grab handle when hovering over the area - probably the area covered by the inserter container in this case - the actual arrows I believe apply a different handle.

Reply:
At the very least, sounds like a good inline code comment.

At best, we ought not need to sprinkle these all around the stylesheets.

Reply:
> At best, we ought not need to sprinkle these all around the stylesheets.

That is correct. So we would need some type of logic to apply the styling when we hover at the different locations - cause there are different stacks if I recall (the ellipsis menu has an area around it and sits above the drag inset. So does the inserter menu I believe). Maybe the approach of having an underlay sitting in the background wasn't ideal after all.